corehq/apps/userreports/sql/adapter.py

```diff
@@ -31,6 +32,12 @@ def drop_table(self):
         with self.engine.begin() as connection:
             self.get_table().drop(connection, checkfirst=True)
 
+    def get_query_object(self):
+        """
+        Get a sqlalchemy query object ready to query this table
+        """
+        return self.sesssion_factory.Session.query(self.get_table())
```

Comment:
Never mind, I see you can do it this way with scoped sessions: http://docs.sqlalchemy.org/en/rel_1_0/orm/contextual.html#implicit-method-access